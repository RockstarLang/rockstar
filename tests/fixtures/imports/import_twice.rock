Say thy Name

Put 1 into the pi
Say "the pi before import is " plus the pi
Featuring library
Say "the pi after import is " with the pi
Put 2 into the pi
Say Foo
Bar taking 541

Say "the pi before 2nd import is " plus the pi
Featuring library
Say "the pi after 2nd import is " plus the pi

Say thy Name