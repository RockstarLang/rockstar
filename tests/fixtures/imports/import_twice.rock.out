
the pi before import is 1
Importing library.rock
Imported library.rock
the pi after import is 3.14159
42
In function library.rock Bar(541)
the pi before 2nd import is 2
Importing library.rock
Imported library.rock
the pi after 2nd import is 3.14159

